.circleci/src/jobs/check.yml

```diff
@@ -4,6 +4,7 @@ steps:
   - checkout
   - check_config
   - restore_yarn_cache
+  - restore_yarn_tools_cache
```

Comment:
Why do we need tools for checks?

Reply:
hmh, though without I got an error, will remove and test again, shouldn't be needed :thinking:

Reply:
Oke, this is needed as ts:check also checks the tools-folder.